### PR TITLE
Add DAC tag to JankStats composable sample

### PR DESCRIPTION
**Pull request by keyboardsurfer**

### CODE REVIEW

**Comment by mlykotom** [comment]
I changed the base branch to merge into the #154 PR to see only the diff from this PR.

File: JankStatsSample/app/src/main/java/com/example/jankstats/compose/ComposeListFragment.kt [hunk 1]
```diff
@@ -71,6 +71,7 @@ class ComposeListFragment : Fragment() {
 @Composable
 fun MessageList(onItemClick: () -> Unit) {
     val listState = rememberLazyListState()
+    // [START compose_jank_metrics]
     val metricsStateHolder = rememberMetricsStateHolder()
```
Comment (mlykotom) [inline JankStatsSample/app/src/main/java/com/example/jankstats/compose/ComposeListFragment.kt:75]: can we add it to the `rememberMetricsStateHolder` as well?

**Review by mlykotom — APPROVED** [review]
Approve with suggestion to add to the `rememberMetricsStateHolder` as well

File: JankStatsSample/app/src/main/java/com/example/jankstats/compose/ComposeListFragment.kt (hunk 1 above)
Comment (keyboardsurfer) [reply JankStatsSample/app/src/main/java/com/example/jankstats/compose/ComposeListFragment.kt:75]: Added

**Pull request merged**